pkgs/servers/pleroma/default.nix

```
@@ -18,21 +18,43 @@
 
 beamPackages.mixRelease rec {
   pname = "pleroma";
-  version = "2.7.0";
+  version = "2.8.0";
 
   src = fetchFromGitLab {
     domain = "git.pleroma.social";
     owner = "pleroma";
     repo = "pleroma";
     rev = "v${version}";
-    sha256 = "sha256-2uKVwjxMLC8jyZWW+ltBRNtOR7RaAb8SPO1iV2wyROc=";
+    sha256 = "sha256-8xhBItGl2BvpYB1N1hZbO1xbtYFldUPCiOI4Kvvywaw=";
   };
 
   patches = [ ./Revert-Config-Restrict-permissions-of-OTP-config.patch ];
 
   mixNixDeps = import ./mix.nix {
     inherit beamPackages lib;
     overrides = final: prev: {
+      phoenix = beamPackages.buildMix {
+        name = "phoenix";
+        version = "v1.7.14-websocket-headers";
```

```suggestion
        version = "1.7.14-websocket-headers";
```
Version must start with a digit. (Unless this policy does not apply to "internal" packages)

---

Not sure if it applies. Let's do that in any case.